verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -700,6 +721,14 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                                                 c.location
                                                     ?.administrativeAreaLevel1,
                                             country: c.location.country,
+                                            latitude: round(
+                                                c.location?.geometry?.latitude,
+                                                4,
+                                            ),
+                                            longitude: round(
+                                                c.location?.geometry?.longitude,
+                                                4,
+                                            ),
```

Comment:
The only thing that gave me pause here was "what if the lat/long fields are not set" (`_.round` returns NaN if the first arg is undefined, and 0 If it is null) but the data don't include any records without latitude or longitude so it's OK. I wonder if it's beneficial to add an assertion here to document that expectation and break if it ever happens. Not worth holding up the review over though, this works well :+1: